### DISCUSSION TITLE
How to change the route of logout?

### DISCUSSION THREAD

**Discussion by azrilaziz** [zacksmash/fortify-ui#30] [Q&A]
Hi, 

I'm curious how to modify/change the route for logout. Currently, the logout redirects to '/'. I would like the logout redirects to '/login'.

Much obliged. Thank you.

**Comment by zacksmash** [comment]
Hey @azrilaziz 

So, this isn't anything specific to `fortify-ui`, however the install process does modify it. In order to change that, you'll want to update the `HOME` constant in this file in your app [`app\Providers\RouteServiceProviders.php`](https://github.com/laravel/laravel/blob/c0b60c0ac74e599122a6a417171798f609f16814/app/Providers/RouteServiceProvider.php#L20).

**Reply by azrilaziz** [reply]
Hi Zack, 

Which means that logout will redirect to the home page. Can I set a custom redirect for the logout route? I would like to differentiate between a home page and a login page.

Thanks in advance.

**Reply by zacksmash** [reply]
Certainly, you can. However that isn't anything that this package would help with. Literally all this does is give you unstyled view templates. Everything else is just plain Laravel and Fortify.
